MAUI/Masked-Entry/Migration.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfMaskedEntry | Syncfusion 
+description: Learn all about Migrating from Syncfusion Xamarin MaskedEdit to Syncfusion .NET MAUI MaskedEntry control and more here.
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---  
+
+# Migrating from Xamarin SfMaskedEdit to .NET MAUI SfMaskedEntry
+
+To migrate easier from `Xamarin SfMaskedEdit` to `.NET MAUI SfMaskedEntry,` most of the APIs were kept from Xamarin SfMaskedEdit in MAUI SfMaskedEntry. However, to maintain the consistency of API naming in MAUI SfMaskedEntry, we renamed some of the APIs. The APIs changed in MAUI SfMaskedEntry from Xamarin SfMaskedEdit are detailed as follows:
```

Comment:
To migrate easier from the `Xamarin SfMaskedEdit` to `.NET MAUI SfMaskedEntry,` most of the APIs are kept from the Xamarin SfMaskedEdit in MAUI SfMaskedEntry. However, to maintain the consistency of API naming in MAUI SfMaskedEntry, rename some of the APIs. The APIs changed in MAUI SfMaskedEntry from Xamarin SfMaskedEdit are detailed as follows.